target/linux/realtek/image/rtl838x.mk

```diff
@@ -27,6 +27,13 @@ define Device/d-link_dgs-1210
 	dlink-version | dlink-headers
 endef
 
+define Device/d-link_dgs-1210-10mp
+  $(Device/d-link_dgs-1210)
+  DEVICE_MODEL := DGS-1210-10MP
+  DEVICE_PACKAGES += lua-rs232
```

Comment:
This used to be included to use the PoE lua script, but a different package is currently being created and submitted for inclusion in OpenWrt. Please drop this package.

Reply:
So this isn't needed for the realtek-poe package?